### PR TITLE
makefile: guarantee runtimetest exists

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -37,6 +37,7 @@ $(RUNTIME_TOOLS_LINK):
 	ln -sf $(CURDIR) $(RUNTIME_TOOLS_LINK)
 
 localvalidation:
+	go build -tags "$(BUILDTAGS)" -o runtimetest ./cmd/runtimetest
```
Comment (mrunalp) [inline Makefile]: We can create a separate target for runtimetest and make localvalidation depend on it.

Comment (Mashimiao) [inline Makefile]: fine, updated.

**Comment by Mashimiao** [comment]
@mrunalp  @liangchenye  @hqhq  PTAL

**Comment by liangchenye** [comment]
LGTM

**Comment by hqhq** [comment]
Needs rebase.

**Comment by Mashimiao** [comment]
Rebased. PTAL @liangchenye  @hqhq

**Comment by liangchenye** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**